docs/Running-Mastodon/Production-guide.md

```diff
@@ -1 +0,0 @@
-[The documentation has moved to its own repository](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md)
```

Comment:
@Gargron understood, I'll delete this fork, close this pull request and make a new one.